pkg/registry/pod/strategy.go

```diff
@@ -231,6 +231,21 @@ func ResourceLocation(getter ResourceGetter, rt http.RoundTripper, ctx api.Conte
 	return loc, rt, nil
 }
 
+func getContainerNames(
+	pod *api.Pod,
+) string {
+	// Collect container names
+	names := ""
+	separator := " "
```

Comment:
This would be better aggregating into a []string and then invoking strings.Join(), less code and more clearly indicates intent